frontend/src/metabase-lib/operators/utils/index.js

```diff
@@ -102,9 +102,9 @@ function populateFields(aggregationOperator, fields) {
   };
 }
 
-export function getAggregationOperators(table) {
+export function getAggregationOperators(table, fields) {
```

Comment:
we used to only get `fields` from `table` however, in the case we have custom fields too, so we need to be able to get `Field` based on those custom fields too.